src/auth/user-record.ts

```diff
@@ -47,8 +47,13 @@ export interface MultiFactorInfoResponse {
   mfaEnrollmentId: string;
   displayName?: string;
   phoneInfo?: string;
+  totpInfo?: TotpInfoResponse;
   enrolledAt?: string;
-  [key: string]: any;
+  [key: string]: unknown;
+}
+
+export interface TotpInfoResponse {
+  [key: string]: unknown;
```

Comment:
Why is this currently empty?
Could you also share an example of how a developer calls the Admin SDK API to obtain this response, please?

Reply:
```
getAuth().getUserByEmail("abc@example.com").then(user => {
    console.log(user.multiFactor);
})
```
Here's an example of how this response is obtained. The user record will display `TotpInfo` in case TOTP has been enrolled by the users. It doesn't cause any changes in creating/updating a user.

Reply:
Thank you!